Automattic/jetpack
# Fixed Jetpack Dashboard submenu highlight when the current page is Settings

Since sites with Nav-unification enabled don't have a Jetpack Settings submenu, we should highlight the Dashboard submenu when the current page is Jetpack > Dashboard > Settings.




Fixes https://github.com/Automattic/wp-calypso/issues/54868

#### Changes proposed in this Pull Request:

* Instead of highlighting the second submenu item from Jetpack menu, we'll highlight the first one since on WoA there's no longer a `Settings` submenu in the Jetpack menu.

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Install Jetpack Beta on your Atomic site.
* Switch to this branch
* Go to Jetpack > Dashboard > Settings
* Make sure that the `Dashboard` submenu item is highlighted instead of the `Activity log`.
* Install Jetpack beta on your Jetpack site (Jurassic)
* Switch to this branch
* Make sure that the `Settings` submenu item is still highlighted
